query-engine/query-engine/src/cli.rs

```diff
@@ -71,7 +71,7 @@ impl CliCommand {
                     query: input.query.clone(),
                     enable_raw_queries: opts.enable_raw_queries,
                     legacy: input.legacy,
-                    datamodel: opts.datamodel(false)?,
+                    datamodel: opts.datamodel(true)?,
```

Comment:
If this is the fix don't we have to apply it for the server as well here? -> https://github.com/prisma/prisma-engines/blob/master/query-engine/query-engine/src/server/mod.rs#L54

Reply:
Good catch.